### PR TITLE
MTP-2394 forsidekort klikkbare, nye farger, fix test

### PR DESCRIPTION
**Pull request by petterkoding**

### CODE REVIEW

File: mt-kit/core/css/src/components/_menu-points.scss [hunk 1]
```diff
@@ -32,12 +40,21 @@
...
+  .mt-h2, .mt-h3 {
     grid-area: heading;
     padding-left: 0;
+    text-decoration: underline;
   }
   p {
     grid-area: keywords;
   }
+
+  &:hover {
```
Comment (lha047) [inline mt-kit/core/css/src/components/_menu-points.scss:52]: The hover and active states seems to repeat. Could you make the code reusable?

**Review by lha047 — CHANGES_REQUESTED** [review]
Looks good. Could you make the state effects reuseable?

File: mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte [hunk 2]
```diff
@@ -12,25 +13,32 @@
...
+          </div>
+        {:else if menuPoint.icon}
+          <img
+            src={menuPoint.icon}
+            alt=""
+            class="mt-img"
+            aria-hidden="true"
+            data-testid="img-icon" />
+        {/if}
+        <p class="mt-{headerTag}">
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte:35]: Maybe we should have header-tag on the titles here

Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte:35]: Maybe add and extra class here so you can target it without listing all the possible header tag? The css will currently not apply for all the possible header-tag options

**Review by lha047 — CHANGES_REQUESTED** [review]
Good job. See comments

File: mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte (hunk 2 above)
Comment (petterkoding) [reply mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte:35]: <h3 class="mt-{headerTag} menu-point-heading"> ?

Comment (lha047) [reply mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte]: Yes 👍

Comment (petterkoding) [reply mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte]: fixed

File: mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte [hunk 3]
```diff
@@ -12,25 +13,32 @@
...
+          </div>
+        {:else if menuPoint.icon}
+          <img
+            src={menuPoint.icon}
+            alt=""
+            class="mt-img"
+            aria-hidden="true"
+            data-testid="img-icon" />
+        {/if}
+        <h3 class="mt-{headerTag} menu-point-heading">
```
Comment (petterkoding) [inline mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte:35]: use HeadingLevel

File: mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte [hunk 4]
```diff
@@ -12,25 +14,32 @@
...
+          </div>
+        {:else if menuPoint.icon}
+          <img
+            src={menuPoint.icon}
+            alt=""
+            class="mt-img"
+            aria-hidden="true"
+            data-testid="img-icon" />
+        {/if}
+        <HeadingLevel class="menu-point-heading" headingLevel={+headerTag.charAt(1)}
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/content/MenuPointsIcons.svelte:36]: Pass mt-h3 for same styling independant of tag

**Review by lha047 — CHANGES_REQUESTED** [review]

**Review by lha047 — APPROVED** [review]
Nice :D

**Pull request merged**